app/app.py

```diff
@@ -109,8 +118,9 @@ def __getattr__(self, name):
 
     server = Server(
         debug=options.debug,
-        keys_path=os.environ.get("TANK_DESKTOP_INTEGRATION_CERTIFICATE", "../resources/keys"),
-        port=os.environ.get("TANK_DESKTOP_INTEGRATION_PORT", 9000)
+        keys_path=os.environ.get("SGTK_BROWSER_INTEGRATION_CERTIFICATE", "../resources/keys"),
+        port=os.environ.get("SGTK_BROWSER_INTEGRATION_PORT", 9000),
+        whitelist=os.environ.get("SGTK_BROWSER_INTEGRATION_WHITELIST")
```

Comment:
Using individual env vars to override feels like overkill when we can use an env var to point at the config file.  Why not just have that and then read these values from there?  Feels easier to maintain.